Readme.md

```diff
@@ -39,6 +39,24 @@ To check the version:
 
 `pip3 uninstall tstbtc` to uninstall the application
 
+## Testing
+
+To run the tests, run the below command:
+
+`cd test`
+
+To run the unit tests 
+
+`pytest -v -m main -s`
+
+To run the feature tests
+
+`pytest -v -m feature -s`
+
+To run the full test suite
+
+`pytest -v -s`
+
```

Comment:
Install pytest (should be added to readme). Ran `pytest -v -s`

<img width="1221" alt="Screen Shot 2023-02-02 at 1 48 59 PM" src="https://user-images.githubusercontent.com/755825/216421505-56087c49-1a0f-40f5-a40d-b92bc60ac10b.png">